lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -936,6 +957,33 @@ void Emitter::emitLibraryPrimTypedByFirstOutputPort(
            << LParen() << bitWidth << RParen() << semicolonEndL();
 }
 
+void Emitter::emitLibraryFloatingPoint(Operation *op) {
+  auto cell = cast<CellInterface>(op);
+  unsigned bitWidth =
+      cell.getOutputPorts()[0].getType().getIntOrFloatBitWidth();
+  unsigned expWidth, sigWidth;
+  if (bitWidth == 16) {
```

Comment:
I know this is the default size for Half, but there are other 16-bit values. Maybe comment saying this is for Half-float?